src/sphinx_ansible_theme/static/css/ansible.css

```
@@ -144,6 +144,10 @@ table.documentation-table .value-required {
 .wy-menu-vertical a.reference.internal {
     padding: 0.4045em 1.618em;
 }
+/* Fix truncation when extranav is hidden */
+.wy-menu-vertical {
```

How about using:
```suggestion
.wy-menu-vertical:not(:has(.sideBanner)) {
```
Then this is only used if there's no `<div class="sideBanner">` inside the `<div class="wy-menu-vertical">`.